xml/cha_administration.xml

```diff
@@ -28,4 +28,184 @@
+      no longer work.
+     </para>
+    </step>
+    <step>
+     <para>
+      Connect the instance to the &suse; update infrastructure in the
+      public cloud. Replace <replaceable>REGISTRATION_CODE</replaceable> with a
+      valid registration code. Replace <replaceable>EMAIL_ADDRESS</replaceable>
+      with the e-mail address associated with the &suse; account you or your
+      organization uses to manage subscriptions.
+     </para>
+     <screen>&prompt.root;<command>registercloudguest</command> -r <replaceable>REGISTRATION_CODE </replaceable> -e <replaceable>EMAIL_ADDRESS</replaceable></screen>
+     <para>
+      This will only register the base product and any recommended products.
+      For instances created from images with a datestamp later than
+      <literal>20220103</literal>, it will also set up the repositories for the
+      <literal>Public Cloud Module</literal>.
+     </para>
+    </step>
+    <step>
+     <para>
+      For instances created from images with a date stamp prior to
+      <literal>20220103</literal> it is strongly recommended to also add the
+      <literal>Public Cloud Module</literal> repositories. For &sle; 15 SP3,
+      run:
+     </para>
+<screen>&prompt.root;<command>SUSEConnect</command> -p sle-module-public-cloud/15.3/x86_64</screen>
+     <para>
+      For other versions or a complete list of modules and their product
+      identifiers, run <command>SUSEConnect --list-extensions</command>.
+     </para>
+    </step>
+   </procedure>
+  </sect2>
+  <sect2 xml:id="sec-admin-suseconnect">
+   <title>Register with <command>SUSEConnect</command></title>
+   <para>
+    To register a BYOS instance against the &scc;, use the command:
+   </para>
+   <screen>&prompt.root;<command>SUSEConnect</command> -r <replaceable>REGISTRATION_CODE</replaceable> -e <replaceable>EMAIL_ADDRESS</replaceable></screen>
+   <para>
+    Replace <replaceable>REGISTRATION_CODE</replaceable> with a valid
+    registration code. Replace <replaceable>EMAIL_ADDRESS</replaceable> with the
+    e-mail address associated with the &suse; account you or your organization
+    uses to manage subscriptions.
+   </para>
+   <para>
+    To register with a local registration server, also provide the URL to
+    the server:
+   </para>
+   <screen>&prompt.root;<command>SUSEConnect -r <replaceable>REGISTRATION_CODE</replaceable> -e <replaceable>EMAIL_ADDRESS</replaceable> --url &quot;<replaceable>URL</replaceable>&quot;</command></screen>
+  </sect2>
+ </sect1>
+
+ <sect1 xml:id="sec-admin-deregister">
+  <title>Deregister instances</title>
+  <para>
+   If you are to decommission an instance, remember to deregister it before
+   termination with:
+  </para>
+  <screen><command>registercloudguest</command> <option>--clean</option></screen>
+  <para>
+   This will ensure that the system gets removed from the &scc; and is no longer
+   counted against your subscription.
+  </para>
+ </sect1>
+
```

Comment:
Re: Troubleshooting, what I'm hearing is that they should run registercloudguest --clean and try again. If that fails, they can try the SUSEConnect method (remembering to de-register appropriately, I imagine.)

We could also point them to https://scc.suse.com/docs/help or other resources to see if they are registered properly.

Reply:
I'm afraid we have to repeat the commands here. Deregistering an instance when it gets decommissioned and reregiering it because of problems are two separate things. Most users these days access our documentation from a search engine, means they search for deregistering or reregistering. We have no control where they land and how much text they actually read, so each section needs to be self-contained.